src/components/UserDetailSections/PatronBlock/PatronBlock.js

```diff
@@ -225,22 +232,25 @@ class PatronBlock extends React.Component {
         }}
       />;
     const title =
-      <Headline size="large" tag="h3">
+      <Headline
+        size="large"
+        tag="h3"
+      >
         <FormattedMessage id="ui-users.settings.patronBlocks" />
         {(hasPatronBlocks) ? <Icon size="medium" icon="exclamation-circle" status="error" /> : ''}
       </Headline>;
 
     return (
       <Accordion
-        open={expanded}
+        data-test-patron-blocks-accordion
         id={accordionId}
+        open={!_.isEmpty(patronBlocks)}
```

Comment:
I am almost sure that current changes will broke the accordion. It will no be toggled. Please recheck it

Reply:
@annamelnyk is correct here; this change causes the accordion to be locked-open when `patronBlocks` is non-empty. I thought we could use the [`<Accordion>` prop `closedByDefault`](https://github.com/folio-org/stripes-components/tree/6087a2c1cb763163950f04b5e70086e1c7116bc1/lib/Accordion) to get what you want here, but I'm not having luck with that either. 

From reading the Jira it seems like there is a larger problem of accordion-state carrying over from one user to the next. Maybe we should put this off until a larger refactor can be done here?